pkg/log/context.go

```diff
@@ -54,3 +57,14 @@ func goid() string {
 	idField := strings.Fields(strings.TrimPrefix(string(buf[:n]), "goroutine "))[0]
 	return idField
 }
+
+func MeasureOperation(operation string, log logrus.FieldLogger, metricsApi metrics.API) func() {
```

Comment:
Why put it in log package?  Seems like a very strange place for it.

Reply:
was meant to log this message and then added metrics for it. So not sure where to add it,

Reply:
maybe common?

Reply:
but we don't have common pkg. Dunno where to add it.